src/index.js

```diff
@@ -277,6 +277,23 @@ export default class HyperScreen extends React.Component {
     );
   }
 
+  /**
+   * Checks if `once` is previously applied and sets `ran-once` if required.
+   */
+  isOncePreviouslyApplied = (behaviorElement) => {
```

Comment:
I think the method name could better describe that it does mutate the element.

Reply:
Difficulty with naming and the fact that this function is both checking and mutating state is a strong sign it should be two separate functions.

Reply:
Addressed in https://github.com/Instawork/hyperview/pull/534/commits/92114e701835884b5b07b862056652cef402c106